index.html

```diff
@@ -2,10 +2,74 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Maithrika Sara Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Old+Standard+TT|VT323" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Maithrika Sara's Portfolio</title>
   </head>
-  
+
   <body>
 
+    <header>
+      <h1>Maithrika Sara</h1>
+
+      <nav class="nav-bar">
+        <div class="icon-menu">
+        </div>
+
+        <ul>
+          <li class="tab" data-content="projects"><a href="#" class="icon-projects"> Projects</a></li>
+          <li class="tab" data-content="about"><a href="#" class="icon-about"> About</a></li>
```

Comment:
Nice work on your nav bar and associating data attributes with sections on your page.